node/ltnodeclient.js

```diff
@@ -93,10 +93,12 @@ function sbRequire(require, resolve, path) {
   if(!getSB(path)) {
     if(path.match(/[\\\/]/) && fpath.extname(path) == ".js") {
       var sb = sandboxes[path.toLowerCase()] = context(path);
```

Comment:
Sorry, to clarify, anyone on the Light Table team can review and merge it.

My comment was simply to suggest it is not likely that any of us will tell you to change the coding style.